src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.csproj

```diff
@@ -7,11 +7,16 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard2.0-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard2.0-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System.Runtime.Intrinsics.X86.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
f the Ref assembly surface area will be the same for netstandard and netcoreapp, I would just suggest to only keep the netsandard one and remove the netcoreapp as there is no point of crosscompiling when netstandard supports netcoreapp.

Reply:
There's probably a way to fix this, but ApiCompat was failing when I did that because `netstandard.dll` couldn't be found (which the contract references if it targets `netstandard`).

Reply:
I suppose you mean ApiCompat on the implementation project? Can you get a repro so that we can check what is wrong? It is possible that in the implementation you need to add a reference to the netstandard facade in order to converge type universe.

Reply:
> a reference to the netstandard facade in order to converge type universe

That's definitely what it was -- I just wasn't sure how to do that. It didn't seem like the netstandard facade was actually built yet during that phase of the build.

Reply:
I think it should be built try just adding `<Reference Include="netstandard">`. That should fix it.

Reply:
> There's probably a way to fix this, but ApiCompat was failing when I did that because netstandard.dll couldn't be found (which the contract references if it targets netstandard).

I tried to keep here same as other packages and that works fine
```xml
  </ItemGroup>
  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
  </ItemGroup>
  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
    <Reference Include="System.Runtime" />
  </ItemGroup>
```

Reply:
```xml
  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
    <Reference Include="System.Runtime" />
  </ItemGroup>
```
That part doesn't seem right. you shouldn't need to depend on System.Runtime contract if you are targeting netstandard. In any case, what we are trying to do is to remove the netcoreapp specific build as I believe it is not requried.

Reply:
> to remove the netcoreapp specific build as I believe it is not requried.

This does not work. But if I just remove the `netstandard` ItemGroup, that works
```xml
  <ItemGroup>
    <Compile Include="System.Runtime.Intrinsics.X86.cs" />
  </ItemGroup>
  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
  </ItemGroup>
  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
</Project>
```

Reply:
that is because you will need to remove also the Configurations.props line that is for netcoreapp. Basically this ref should only ever build for netstandard.